src/openrct2/network/NetworkBase.cpp

```diff
@@ -43,7 +43,7 @@
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "4"
```

Comment:
This needs rebasing and re-incrementing. This doesn't show up as a merge conflict, because current develop has the exact same change.